src/Cache.js

```diff
@@ -7,57 +7,42 @@ const decode = (value) => {
   return value
 }
 
-// Keys are namespaced by db name, since in orbit each store gets its own cache
-const keyWrap = (prefix, key) => `${prefix}_${key}`
+// Maps keys from orbit-db v22 to keys currently in our implementation
+const keyMap = (prefix, key) => {
+  const splits = key.split('/')
+  const heads = key.includes('Heads')
+  return `${splits[2]}/${splits[3]}_${heads ? splits[4] : key}`
+}
 
 class Cache {
-  constructor (storage, dbAddress, opts) {
-    this.opts = opts
-    this.store = storage
-    this.dbAddress = dbAddress
+  constructor (redisOpts) {
+    this.redisOpts = redisOpts
+    this.store = redis.createClient({host:'redis'})
   }
 
-  async open () { }
+  open () {
+    return Promise.resolve()
+  }
 
   async close () { }
 
-  async destroy () {
-    // delete all keys ?
-  }
+  async destroy () { }
 
   get (key) {
     return new Promise((resolve, reject) => {
-      this.store.get(keyWrap(this.dbAddress, key), (err, reply) => {
+      this.store.get(keyMap(this.dbAddress, key), (err, reply) => {
```

Comment:
Where does `this.odbAddress` come from? I see that it was removed from the constructor.